bluebrain/repo-bluebrain/packages/py-atldld/package.py

```diff
@@ -30,9 +29,24 @@ class PyAtldld(PythonPackage):
     depends_on("py-dataclasses", when="@0.3.1: ^python@3.6", type=("build", "run"))
     depends_on("py-matplotlib", type=("build", "run"))
     depends_on("py-numpy", type=("build", "run"))
-    depends_on("py-opencv-python", type=("build", "run"))
+    depends_on("opencv+python3+python_bindings_generator+imgproc", type=("build", "run"))
     depends_on("py-pandas", type=("build", "run"))
     depends_on("py-pillow", when="@0.3.1:", type=("build", "run"))
     depends_on("py-requests", type=("build", "run"))
     depends_on("py-responses", type=("build", "run"))
     depends_on("py-scikit-image", type=("build", "run"))
+
+    def setup_run_environment(self, env):
```

Comment:
Do we really need this? Shouldn't the dependency on `py-numpy` do this automatically?

Reply:
Although I guess that's more for modules and not `spack load` or environments :thinking:

Reply:
I tried in an environment and this was the magic that made it work for me. Which is no guarantee that I was doing everything correctly.

Reply:
Environment on bb5? This is rather hackish, since `intel-oneapi-mkl` fills in for a provider, there's no guarantee that this will work in general.

Let's remove it for now and figure out if the module still works? That's sort of the penultimate test here, anyways